### PR TITLE
Fix wrong variable reference in Grafana & Pulsar Manager port

### LINKED ISSUE #40: Error: unable to build kubernetes objects from release manifest - Validation Error
**Describe the bug**
Consistently get the following error stack when running **_helm install_**

`Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Deployment.spec.template.spec.containers[0].ports[0]): missing required field "containerPort" in io.k8s.api.core.v1.ContainerPort`

**Assumptions**
1. k8s setup
2. HELM available

**To Reproduce**
1. git clone https://github.com/apache/pulsar-helm-chart.git
2. git checkout master
3. cd pulsar-helm-chart
4. scripts/pulsar/prepare_helm_release.sh -n pulsar -c
5. helm install --generate-name --values=$PWD/examples/values-one-node.yaml charts/pulsar

**Expected behavior**
`NAME: pulsar-********
LAST DEPLOYED: *****
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None`

**Screenshots**
pulsar-helm-error

**Desktop (please complete the following information):**
 - macOS - 10.15.5
 - Docker Desktop - 2.3.0.3
 - Kubernetes - v1.16.5
 - HELM - v3.1.1

### PR DESCRIPTION
**Pull request by NiklasWagner**
Fixes #40 

### Motivation

PR #37 updated the location of the ports in the default values yaml. This causes a null pointer exception when rendering this helm chart.

### Modifications

Fix variable reference

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### CODE REVIEW

**Comment by rvashishth** [comment]
Thanks @NiklasWagne

**Comment by NiklasWagner** [comment]
Weird that the CI didn't throw an error on your old PR. This should've be definitely catched by the CI.

File: charts/pulsar/templates/pulsar-manager-deployment.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ spec:
 {{ toYaml .Values.pulsar_manager.resources | indent 12 }}
         {{- end }}
           ports:
-          - containerPort: {{ .Values.pulsar_manager.port }}
+          - containerPort: {{ .Values.pulsar_manager.service.port }}
```
Comment (rvashishth) [inline charts/pulsar/templates/pulsar-manager-deployment.yaml:61]: I assume we should use `targetPort` here, as the `targetPort` under service is for pods.

Comment (NiklasWagner) [reply charts/pulsar/templates/pulsar-manager-deployment.yaml]: Yep you're correct! Reading this here: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
```yaml
apiVersion: v1
kind: Service
metadata:
  name: my-service
spec:
  selector:
    app: MyApp
  ports:
    - protocol: TCP
      port: 80
      targetPort: 9376
```

>This specification creates a new Service object named “my-service”, which targets TCP port 9376 on any Pod with the app=MyApp label.

**Pull request merged**